### PR TITLE
194 support for traffic models

### PR DESCRIPTION
**Pull request by wybert**
Should solve #194

### CODE REVIEW

File: knime_extension/src/nodes/spatialnetwork.py [hunk 1]
```diff
@@ -247,6 +270,7 @@ class GoogleDistanceMatrix:
         description="""Click [here](https://developers.google.com/maps/documentation/distance-matrix/get-api-key) for details on
         how to obtain and use a Google API key for the Distance Matrix API.""",
         default_value="",
+        # TODO: Check that API key is present
```
Comment (koettert) [inline knime_extension/src/nodes/spatialnetwork.py:273]: See todo

File: knime_extension/src/nodes/spatialnetwork.py [hunk 2]
```diff
@@ -289,6 +345,7 @@ def fetch_google_od(download_link):
             nt_duration = 0
             nt_distance = 0
             try:
+                # TODO: Add advanced timeout parameter
```
Comment (koettert) [inline knime_extension/src/nodes/spatialnetwork.py:348]: See todo

**Review by koettert — CHANGES_REQUESTED** [review]
Shouldn't the node return the duration_in_traffic column in addition to the duration column if the consider traffic option is enabled? It seems that the duration field returns the default duration and the duration_in_traffic contains the duration when considering the traffic

File: knime_extension/src/nodes/spatialnetwork.py (hunk 2 above)
Comment (koettert) [reply knime_extension/src/nodes/spatialnetwork.py:348]: Please also change the library to the more common request library

File: knime_extension/src/nodes/spatialnetwork.py (hunk 1 above)
Comment (UrbanGISer) [reply knime_extension/src/nodes/spatialnetwork.py:273]: might be hard to check whether the key is valid

File: knime_extension/src/nodes/spatialnetwork.py (hunk 2 above)
Comment (UrbanGISer) [reply knime_extension/src/nodes/spatialnetwork.py]: library changed

**Pull request merged**